proto/sound_sensor.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+package smartcore.bos;
+
+option go_package = "github.com/vanti-dev/sc-bos/pkg/gen";
+
+import "google/protobuf/field_mask.proto";
+import "google/protobuf/timestamp.proto";
+import "types/info.proto";
+
+// Trait for devices that measure sound/noise levels.
+service SoundSensorApi {
+  rpc GetSoundLevel(GetSoundLevelRequest) returns (SoundLevel) {}
+  rpc PullSoundLevel(PullSoundLevelRequest) returns (stream PullSoundLevelResponse) {}
+}
+
+service SoundSensorInfo {
+  rpc DescribeSoundLevel(DescribeSoundLevelRequest) returns (SoundLevelSupport) {}
+}
+
+// SoundLevel represents a sound/noise level.
+message SoundLevel {
+  // The measured sound/noise level, usually in dBA or dB. Use Info service to check.
+  // Optional. 0 means no sound or possibly broken sensor. Omitted means unknown.
+  optional float sound_pressure_level = 1;
+}
+
+message SoundLevelSupport {
+  // How a named device supports read/write/pull apis
+  smartcore.types.ResourceSupport resource_support = 1;
+  // The unit associated with the sound_level value in dBA or dB.
```

Comment:
The referenced property is named `sound_pressure_level`. 
Comment is phrased in an unclear way. Suggestion:

    // The unit associated with the sound_pressure_level value.
    // Known units:
    //   - dBA - decibels SPL, A-weighted
    //   - dB - decibels SPL, unweighted